File: lab1/src/objects/SortMyDot.java
package objects;

import java.util.Comparator;

import kitpo.IUserType;

public class SortMyDot implements  Comparator<MyDot> {
    @Override
    public int compare(MyDot  o1, MyDot  o2) {
        return o1.getRange().compareTo(o2.getRange());
    }
}

File: lab3/src/main/kotlin/Objects/SortMyDot.kt
package Objects

import List.IUserType


class SortMyDot : Comparator<IUserType> {
    override fun compare(o1: IUserType, o2: IUserType): Int {
        o1 as MyDot
        o2 as MyDot
        return o1.range.compareTo(o2.range)
    }
}